deploy/charts/harvester/README.md

```diff
@@ -12,51 +12,54 @@ This chart will do the following:
 - Deploy a CDI CRD resource to enable KubeVirt Containerized Data Importer(CDI) if needed, default to deploy.
 - Deploy a Minio as virtual machine image storage if needed, defaults to deploy.
 - Deploy the Harvester resources.
+- Longhorn as build-in storage management(enable by set `longhorn.enabled=true` in the helm install).
+- [Multus-CNI](https://github.com/intel/multus-cni) as default multi networks management solution(enable by set `multus.enabled=true` in the helm install).
 
 ### Prerequisites
 
 - Kubernetes 1.16+.
 - Helm 3.2+.
-- **Default** [StorageClass](https://v1-16.docs.kubernetes.io/docs/concepts/storage/storage-classes/).
-
-#### Notes
-
-- Use the existing KubeVirt/CDI/Image-Storage Service.
-
-    If you have already prepared the KubeVirt, CDI or image storage compatible with S3 like Minio, you can disable these installations in this chart.
-    
-    ```bash
-    $ helm install harvester harvester --namespace harvester-system \
-        --set kubevirt.enabled=false --set kubevirt-operator.enabled=false \
-        --set cdi.enabled=false --set cdi-operator.enabled=false \
-        --set minio.enabled=false
-    ```
-
-- Change the default StorageClass.
-    
-    By default, the default StorageClass is required for both the Minio and [`DataVolume`](https://github.com/kubevirt/containerized-data-importer#datavolumes). You can follow the [official document](https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/) to change the default StorageClass of Kubernetes cluster.
+- **Default** [StorageClass](https://v1-16.docs.kubernetes.io/docs/concepts/storage/storage-classes/) if the Longhorn is not installed.
```

Comment:
This sounds like users need to deploy Longhorn separately.
What about `if the Longhorn in harvester chart is disabled`